gitops/README.md

```diff
@@ -41,18 +57,57 @@ b. Merge a PR with the kubeconfigs and overlay kustomization to trigger a Pipeli
     KCP_ORG="root:pipelines-service" KCP_WORKSPACE="compute" DATA_DIR="/home/workspace/pipelines-service" ../images/kcp-registrar/register.sh
     ```
 
-2. Open and merge a Pull Request
+2. Setup PaC and merge a Pull Request
 There are a couple of steps that need to be done to use PAC in Pipelines-service.
 a. Install Pipelines as Code on the OpenShift Cluster.
+   - Following the instructions from [here](https://pipelinesascode.com/docs/install/installation/):
+     ```
+     kubectl patch tektonconfig config --type="merge" -p '{"spec": {"addon":{"enablePipelinesAsCode": false}}}'
+     kubectl apply -f https://raw.githubusercontent.com/openshift-pipelines/pipelines-as-code/stable/release.yaml
+     ```
 b. Set up a Git application on any of the Git providers (GitHub, GitLab, Bitbucket etc.) by following the steps from [here](https://pipelinesascode.com).
+   - As an example, if you wish to set up a GitHub app, we recommend using the [tkn pac cli tool](https://pipelinesascode.com/docs/guide/cli/) as it is easy to complete the whole process in just a couple of commands.
+   ```
+   tkn pac bootstrap github-app
+      This command will guide you through the process to create a GitHub app. 
+   tkn pac create repo
+      This command will create a new Pipelines as Code Repository definition, a namespace where the pipelineruns run and configures webhook.
+   ```
+   - Do note that you need to install the GitHub app on your desired repository apart from the above two steps.
+   - PaC also supports the above steps on other Git providers such as Gitlab, Bitbucket all of which is documented [here](https://pipelinesascode.com/).
 
 Once the above two steps are done, the GitHub app (or any other Git provider app) you've set up from step 2 is now talking to PAC and would watch PRs and trigger PipelineRuns that automate the workflow previously described.
 
-The PR should have:
+In order to generate the kubeconfig for the cluster and kcp with the appropriate permissions using service accounts, we have created few scripts which are available [here](../images/access_setup/content/bin). The [README.md](../images/access_setup/README.md) describes the steps but essentially, you need the below two commands to generate kubeconfig for your cluster and kcp instance respectively.
+```
```

Comment:
This section contains steps that are also useful if running the setup manually.
I would that info earlier. Maybe we need subsections "### Prepare repository" and "### Deploy repository"?